apisix/plugins/openid-connect.lua

```diff
@@ -148,6 +150,13 @@ end
 
 function _M.rewrite(plugin_conf, ctx)
     local conf = core.table.clone(plugin_conf)
+
+    -- Previously, we multiply conf.timeout before storing it in etcd.
+    -- If the timeout is too large, we should not multiply it again.
+    if not (conf.timeout >= 1000 and conf.timeout % 1000 == 0) then
```

Comment:
Hi @spacewander 👋 May I know why we need to use `multiply` here? 🤔 because both our document[1] and code comments (`schema.description`) says it's in `second`.

[1] https://apisix.apache.org/docs/apisix/next/plugins/openid-connect